lib/seafoam/commands.rb

```diff
@@ -451,6 +452,8 @@ def render(name, *args)
           pass_options[:simplify_truffle_args] = false
         when '--no-simplify-alloc'
           pass_options[:simplify_alloc] = false
+        when '--show-null-fields'
+          pass_options[:hide_null_fields] = false
```

Comment:
In another PR we should add some kind of `--no-transformations` option to turn everything off.